src/command/dns-command.ts

```diff
@@ -41,10 +77,11 @@ export const dnsCmd = async (options: DnsOptions): Promise<DnsQueryResult> => {
 		'-4',
 		'+time=1',
```

Comment:
The official manual has this parameter as `timeout`. So to be safe lets use that and increase it too `+timeout=3`

Reply:
done